manifests/params.pp

```diff
@@ -20,20 +21,39 @@
   $rcfile                = "${::root_home}/.mongorc.js"
   $dbpath_fix            = false
 
-  $mongos_service_manage = pick($mongodb::globals::mongos_service_manage, true)
-  $mongos_service_enable = pick($mongodb::globals::mongos_service_enable, true)
-  $mongos_service_ensure = pick($mongodb::globals::mongos_service_ensure, 'running')
-  $mongos_service_status = $mongodb::globals::mongos_service_status
-  $mongos_configdb       = '127.0.0.1:27019'
-  $mongos_restart        = true
+  $mongos_service_manage     = pick($mongodb::globals::mongos_service_manage, true)
```

Comment:
Please don't realign these. It creates a lot of noise in changelogs

Reply:
My bad, I think I had noticed puppet lint or one of the tests in the repo complaining about alignment. I'll avoid anymore unnecessary alignment changes.